plugins/octopus-deploy/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@backstage/plugin-octopus-deploy",
+  "version": "0.1.0",
```

Comment:
it's fine to start at 0.0.0 (so with the changeset, the first released version becomes 0.1.0)

Reply:
Fixed :)